presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStats.java

```diff
@@ -50,7 +52,8 @@
             long planNodeOutputPositions,
             DataSize planNodeOutputDataSize,
             Map<String, OperatorInputStats> operatorInputStats,
-            Map<String, OperatorHashCollisionsStats> operatorHashCollisionsStats)
+            Map<String, OperatorHashCollisionsStats> operatorHashCollisionsStats,
+            Optional<WindowNodeStats> windowNodeStats)
```

Comment:
Addressed in f094637 (kept as a fixup, so you could continue with remaining part of the review and then confirming that the addressed part looks good too).